packages/db/fauna/resources/Index/pinsSizes.js

```diff
@@ -0,0 +1,38 @@
+import fauna from 'faunadb'
+
+const {
+  CreateIndex,
+  Collection,
+  If,
+  Index,
+  Exists,
+  Not
+} = fauna
+
+/**
+ * Usage:
+ *
+ * Match(
+ *   Index('pins_sizes'),
+ *   StatusString // Pinned, Pinning, PinQueued, ...
+ * )
+ */
+const index = {
+  name: 'pins_sizes',
+  source: [
+    Collection('Pin')
+  ],
+  terms: [
+    { field: ['data', 'status'] }
+  ],
+  values: [
+    { field: ['data', 'content', 'data', 'dagSize'] }
```

Comment:
I think this is not working.

Reply:
you are right! Before I added the status condition it seemed to be working. I will look into it

Reply:
I think you can't traverse a relation.

Reply:
I think this was working for a weird combination of merging the index updates in Fauna as I know cannot make it work lol

Reply:
Yes, I am working on a new index/query for this

Reply:
![image](https://user-images.githubusercontent.com/7295071/125476025-54eea123-97da-4c27-afb0-28735e1e3350.png)
Should now be fixed. I changed the index logic to get all the pins content ref and then sum their dagSize. Not sure if this is the most efficient though